content/v1.11/software/install.md

```diff
@@ -239,6 +252,30 @@ crossplane-stable/crossplane \
 -f settings.yaml
 ```
 
+#### Feature flags
+
+Crossplane introduces new features behind feature flags. By default
+alpha features are off. Crossplane enables beta features by default. To enable a 
+feature flag, set the `args` value in the Helm chart. Available feature flags
+can be directly found by running `crossplane core start --help`, or by looking 
+at the table below.
+
+{{< expand "Feature flags" >}}
+{{< table caption="Feature flags" >}}
+| Status | Flag | Description |
+| --- | --- | --- |
+| Beta | `--enable-composition-revisions` |Enable support for CompositionRevisions |
+| Alpha | `--enable-composition-functions` | Enable support for Composition Functions. |
+| Alpha | `--enable-composition-webhook-schema-validation` | Enable Composition validation using schemas. |
+| Alpha | `--enable-environment-configs` | Enable support for EnvironmentConfigs. |
+| Alpha | `--enable-external-secret-stores` | Enable support for External Secret Stores. |
+{{< /table >}}
+{{< /expand >}}
+
+Set these flags either in the `values.yaml` file or at install time using the
+`--set` flag, for example: `--set
+args=["--enable-composition-functions","--enable-composition-webhook-schema-validation"]`.
```

Comment:
Should be, here and above at https://github.com/crossplane/docs/pull/459/files#diff-866251fa733e9d39611b10e98240316b445f0e87f4096b3d12b72e8d0303452aR277 :
```suggestion
args={"--enable-composition-functions","--enable-composition-webhook-schema-validation"}`.
```